QModManager/Patching/ManifestValidator.cs

```diff
@@ -18,6 +18,8 @@ internal class ManifestValidator : IManifestValidator
             { "QModManager", ModStatus.BannedID },
             { "QModInstaller", ModStatus.BannedID },
             { "EnableAchievements", ModStatus.Merged },
+            { "AgonyAssetTools", ModStatus.Obsolete },
```

Comment:
If we find ourselves in the future needing to change this list very often, we can use a system similar to the version check to get the list from github and cache it in a file.

Reply:
It is my deepest hope that we never get to a point of needing to update this list frequently.